insights/terraform/projects/bfd/api-requests-beneficiaries/glue.tf

```diff
@@ -4,14 +4,17 @@
 
 # Glue Catalog Table to hold Beneficiaries
 module "beneficiaries-table" {
```

Comment:
minor thing that is not from this PR but I notice that the module and resource names for the tables are still on the old conventions.

Reply:
Yeah, that's a good point. I was pretty careful with the AWS resource names, but I left a lot of the terraform resource names alone. I'm of the opinion that we don't need to name every terraform resource / module `bfd-insights-bfd-<environment>-<resource>`, because they're local and shouldn't really ever be seen outside of the context of either this part of the codebase or maybe if you were delving the terraform lock file. (Also, you couldn't include the environment in the name of the resource, because that's derived from the workspace).

That said, I think it might be a good idea to standardize some of those into maybe an abbreviated convention. It was more apparent when we were using `aws_glue_catalog_table.beneficiaries-table` but now that it's `module.beneficiaries-table`, it's not so clear). So maybe something like general function + resource identifier: `glue-table-beneficiaries` for this one?

Reply:
I standardized the terraform resource names. I'm considering also revisiting some of the AWS resource identifiers, but if I do, I'll leave it for BFD-1741, because after that ticket, it will be hard to revisit names.